src/treebuilders/map.cpp

```diff
@@ -66,12 +66,12 @@ namespace mrcpp {
  *
  */
 template <int D>
-void map(double prec, FunctionTree<D> &out, FunctionTree<D> &inp, FMap fmap, int maxIter, bool absPrec) {
+void map(double prec, FunctionTree<D, double> &out, FunctionTree<D, double> &inp, FMap<double, double> fmap, int maxIter, bool absPrec) {
```

Comment:
is `map` not relevant when one has `float` or `std::complex` scalar types?

Reply:
I don't know, but since it is not used I prefer to not implement before we know what we want to use it for.